src/ws_corretion.py

```diff
@@ -0,0 +1,298 @@
+import time
```

Comment:
This file should be called 'ws_correction.py'

Reply:
Thanks good catch!